### DISCUSSION TITLE
How do I style an entire cell based on its value?

### DISCUSSION THREAD

**Discussion by FBosler** [TanStack/table#2665] [General]
It seems like this problem has been answered for [v6](https://codesandbox.io/s/744n0nmvl0?file=/index.js:552-985) but does not work for v7 any longer. Any chance somebody could point me into the right direction?

**Comment by gargroh** [comment]
With v7 dom is in your control, you can modify styles of Cell based on that

**Reply by FBosler** [reply]
@gargroh thx for the reply! I've just started using react-table so I am wondering if there is an example for my use-case that is deemed best practice? 

I think there should be one and preferably only one best way of doing something. :)

**Comment by dkulakov1** [comment]
Hey, look on this example
https://react-table.tanstack.com/docs/examples/data-driven-classes-and-styles
